openapi.yml

```diff
@@ -658,6 +669,36 @@ components:
           "registration_status": "PRE_DRAFT",
           "scope": "NATIONAL"
         }
+    OrganizationResult:
+      allOf:
+        - $ref: "#/components/schemas/Organization"
+        - type: object
+          properties:
+            links:
```

Comment:
Then the whole `OrganizationResult` schema can be deleted, because after removing this property it is an exact copy of `Organization`, right?

Reply:
yes indeed. just curios, why did you use this links property ?

Reply:
It is used in order to decouple the backend and the frontend logic, according to the [HATEOS](https://en.wikipedia.org/wiki/HATEOAS) principle. The basic idea is that the frontend must not have in advance the knowledge of the allowed operations on a resource, but the backend will embed such information in the resource itself returned to the client, so that the client is driven in the workflow by the server.

Reply:
Since AFAIK we don't plan to use the client that way (with some autodiscovery mechanisms) I'd keep hateoas apart for the moment